app/Support/Version.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Support;
+
+use App\Enums\VersionEnum;
+use Illuminate\Support\Facades\Route;
+
+class Version
```

Comment:
let's also move this class to the LaravelHelpers package and use VersionEnumContract instead of VersionEnum